packages/cli/package.json

```diff
@@ -170,5 +171,8 @@
     }
   },
   "copyright.owner": "IBM Corp.",
-  "license": "MIT"
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  }
```

Comment:
Similarly here, let's move `copyright.owner`, `license` and `publishConfig` closer to the top.